internal/kafka/internal/api/public/model_region_capacity_list_item.go

```diff
@@ -15,7 +15,7 @@ type RegionCapacityListItem struct {
 	InstanceType string `json:"instance_type"`
 	// flag indicating whether the capacity for the instance type in the region is reached
 	// Deprecated
-	DeprecatedMaxCapacityReached bool `json:"max_capacity_reached"`
+	DeprecatedMaxCapacityReached bool `json:"max_capacity_reached,omitempty"`
```

Comment:
Just note that because it's `omitempty` here, if `max_capacity_reached` was *false*, it won't show up in the response. 

Example:
```
"capacity": [
                {
                    "instance_type": "standard",
                    "available_sizes": [
                        "x1"
                    ]
                },
                {
                    "instance_type": "developer",
                    "available_sizes": [
                        "x1"
                    ]
                }
            ]
```

Is that ok with you?

Reply:
Yes. No one will use it and sdks will filter out deprecated fields. On backend we still need those as users might choose to not update instantly to latest SDKs